### PR TITLE
Fixing header names for case-sensitive MinGW

### PR DESCRIPTION
**Pull request by procedural**
Fixes compile time errors on MinGW.

### CODE REVIEW

File: libs/openFrameworks/video/ofDirectShowPlayer.cpp [hunk 1]
```diff
@@ -14,15 +14,14 @@
...
 
 
-#include <DShow.h>
+#include <dshow.h>
 #pragma include_alias( "dxtrans.h", "qedit.h" )
 #define __IDxtCompositor_INTERFACE_DEFINED__
 #define __IDxtAlphaSetter_INTERFACE_DEFINED__
 #define __IDxtJpeg_INTERFACE_DEFINED__
 #define __IDxtKey_INTERFACE_DEFINED__
-#include <uuids.h>
```
Comment (procedural) [inline libs/openFrameworks/video/ofDirectShowPlayer.cpp]: If someone will be curious, [uuids.h is included in dshow.h already](https://github.com/ofTheo/videoInput/blob/master/videoInputSrcAndDemos/libs/DShow/Include/dshow.h#L51)

**Comment by bilderbuchi** [comment]
btw @arturoc on AppVeyor, could you also build codeblocks/MingW, or only VS solutions?

**Comment by bilderbuchi** [comment]
@procedural did you test if this works on codeblocks? also on visual studio?

**Comment by procedural** [comment]
I've tested it and it works on Visual Studio Community 2013 and MinGW-w64 compiler. I use my own build system and don't know what's the state of configs / makefiles in openFrameworks, but this PR fixes compile time errors which are universal for all build systems out there :)

**Comment by bilderbuchi** [comment]
> btw @arturoc on AppVeyor, could you also build codeblocks/MingW, or only VS solutions?

to answer my own question, [apparently you can](https://github.com/cppformat/cppformat/blob/master/appveyor.yml) - would be great to enable that, no?

**Comment by ofTheo** [comment]
I think we might need additional headers and libs for CB for the player to work. 
But this PR is needed regardless so going to Merge. 

Thanks @procedural !

**Pull request merged**